library/spdm_common_lib/libspdm_com_support.c

```diff
@@ -6,6 +6,10 @@
 
 #include "internal/libspdm_common_lib.h"
 
+#if LIBSPDM_FIPS_MODE
+#define LIBSPDM_MODULE_VERSION_NUMBER 0x01
```

Comment:
This needs to be available to the Integrator at compile time, so it will need to reside in `include/library/spdm_common_lib.h`.